src/contexts/AddItemFormContext.jsx

```diff
@@ -0,0 +1,31 @@
+import { createContext, useContext, useState } from 'react';
+
+export const AddItemFormContext = createContext();
+
+export function AddItemFormProvider({ children }) {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+  const [price, setPrice] = useState('');
+  const [tags, setTags] = useState([]);
+
+  const isFormValid = name && description && price && tags.length > 0;
+
+  return (
+    <AddItemFormContext.Provider
+      value={{
+        name,
+        setName,
+        description,
+        setDescription,
+        price,
+        setPrice,
+        tags,
+        setTags,
+        isFormValid,
+      }}>
+      {children}
+    </AddItemFormContext.Provider>
+  );
+}
+
+export const useAddItemForm = () => useContext(AddItemFormContext);
```

Comment:
💊 제안
해당 context를 사용하는 훅을 정의하셔서 사용성을 높으신 것 좋습니다.
다만 아래처럼 context 내부에서 해당 context에 접근하지 않을때의 에러도 추가해주시면 더 좋을 것 같습니다!

```suggestion
export const useAddItemForm = () => {
  const _context = useContext(AddItemFormContext);

  if (!_context) throw new Error(/** error msg */);
  return _context;
}
```